packages/bp/e2e/studio/flows.test.ts

```diff
@@ -16,12 +16,15 @@ describe('Studio - Flows', () => {
   it('Create new flow', async () => {
     await clickOn('#btn-add-flow')
     await fillField('#input-flow-name', 'test_flow')
-    await Promise.all([expectStudioApiCallSuccess('flows'), clickOn('#btn-submit')])
+
+    await clickOn('#btn-submit')
+    await expectStudioApiCallSuccess('flows')
```

Comment:
I think I changed those back in the days because sometimes the button was clicked and the api call was made before the expect was waiting, but i'm not so sure. Promise.all would wait until both are executed, and ensure that the api call is expected before the action to call it

Reply:
That was my guess too. So far I have only had an issue at one place in the code-editor modules tests, but otherwise, it seems to work fine without the Promise.all. I'll look to see if there is something already built-in around puppeteer to manage this.